editor/src/components/canvas/canvas-strategies/canvas-strategies.tsx

```diff
@@ -389,14 +404,9 @@ export function getApplicableControls(
 }
 
 export function isResizableStrategy(canvasStrategy: CanvasStrategy): boolean {
-  switch (canvasStrategy.id) {
-    case 'ABSOLUTE_RESIZE_BOUNDING_BOX':
-    case 'KEYBOARD_ABSOLUTE_RESIZE':
-    case 'FLEX_RESIZE_BASIC':
-      return true
-    default:
-      return false
-  }
+  return ['ABSOLUTE_RESIZE_BOUNDING_BOX', 'KEYBOARD_ABSOLUTE_RESIZE', 'FLEX_RESIZE_BASIC'].some(
```

Comment:
This feels a little fragile, I wonder if it would be better to attach a flag to the strategy itself.

Reply:
This code was to support resizing via the Ancestor meta strategy, which we have since decided is undesirable, so I'll revert it. I do agree though that maybe this is something that should be handled in a different way in general (either via a flag or perhaps by checking that it was produced by a certain meta strategy)